pkg/systemd/terminal.jsx

```diff
@@ -33,7 +33,7 @@ const _ = cockpit.gettext;
                 environ: [
                     "TERM=xterm-256color",
                 ],
-                directory: user.home || "/",
+                directory: cockpit.location.options.path || user.home || "/",
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19116-20230726-231132-ce356f2b-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)